exercises/Ex2LeapYear.js

```diff
@@ -0,0 +1,10 @@
+function isLeapYear(inputYear){
+    if((inputYear%400 == 0)||(inputYear%100 !=0 && inputYear%4 == 0)){
+        var laepOrNo = "leap";
```

Comment:
can use true/false ("leap", "not leap" need more effort when you utilise the result of this method - you basically need to know the result set syntax).